_schema.graphql

```diff
@@ -3477,6 +3459,20 @@ type FilterSaleArtworksCounts {
   ): FormattedNumber
 }
 
+input fixFailedPaymentInput {
```

Comment:
why is this lower case?

Reply:
interesting, so is type `fixFailedPaymentPayload`

Reply:
It was lowercase because I made a typo 😞